packages/fuels-accounts/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use crate::{accounts_utils::extract_message_id, provider::Provider};
 mod accounts_utils;
 pub mod predicate;
 pub mod provider;
+pub mod supported_versions;
```

Comment:
IMO we should not expose this outside of the crate.
```suggestion
mod supported_versions;
```